cypress/e2e/24-ndv-paired-item.cy.ts

```diff
@@ -59,7 +59,8 @@ describe('NDV', () => {
 		ndv.getters.inputTableRow(4).invoke('attr', 'data-test-id').should('equal', 'hovering-item');
 	});
 
-	it('maps paired input and output items based on selected input node', () => {
+	// eslint-disable-next-line n8n-local-rules/no-skipped-tests
```

Comment:
we added this rule people adding `.skip`. 
This is now the 4th test that we have skipped, and added a `eslint-disable` for.

Maybe we should just drop the rule, since it doesn't seem be doing what we wanted it to do.